cloud/scope/cluster.go

```diff
@@ -117,6 +117,34 @@ func (s *ClusterScope) PublicIPSpecs() []azure.PublicIPSpec {
 	}
 }
 
+// LBSpecs returns the load balancer specs.
+func (s *ClusterScope) LBSpecs() []azure.LBSpec {
+	return []azure.LBSpec{
+		{
+			// Public API Server LB
```

Comment:
Is this internal or public? The role says `infrav1.InternalRole` and the comment says Public API Server.

Reply:
that's a bug... great catch 

Should be public

Reply:
actually just the comment is wrong